### PR TITLE
harmonize joint_states subscription

### PR DESCRIPTION
**Pull request by fmessmer**
minor
just found this heterogeneous handling while checking something different

### CODE REVIEW

File: cob_script_server/src/simple_script_server/simple_script_server.py [hunk 1]
```diff
@@ -654,12 +654,13 @@ def compose_trajectory(self, component_name, parameter_name, speed_factor=1.0, u
 		# get current pos
 		timeout = 3.0
 		try:
-			joint_state = rospy.wait_for_message("/" + component_name + "/joint_states", JointState, timeout=timeout)  # type: JointState
+			joint_state_topic = rospy.get_param(self.ns_global_prefix + "/" + component_name + "/joint_state_topic", "/" + component_name + "/joint_states")
```
Comment (floweisshardt) [inline cob_script_server/src/simple_script_server/simple_script_server.py:657]: why is this needed?`convention is that each component has it's own private joint_states. See suggestion in https://github.com/mojin-robotics/mojin_robots/pull/334#issuecomment-790500868

**Review by benmaidel — APPROVED** [review]

File: cob_script_server/src/simple_script_server/simple_script_server.py (hunk 1 above)
Comment (fmessmer) [reply cob_script_server/src/simple_script_server/simple_script_server.py:657]: more details in https://github.com/mojin-robotics/mojin_robots/pull/334
